src/prasad/EmployeeBasicTable.java

```diff
@@ -0,0 +1,194 @@
+package prasad;
+
+import java.awt.List;
+import java.util.LinkedHashSet;
+import java.util.Scanner;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+/*Assignment 9:-*/
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class EmployeeBasicTable {
+	String fName = "Dhara";
+	String tableHeader = "//table[@id='table1']/thead/tr/th";
```

Comment:
We don't write xpath as instance variable. this should be part of respective method